client/src/app/containers/Game/ScoreBoard.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+import blue from '@material-ui/core/colors/blue';
+import grey from '@material-ui/core/colors/grey';
+import green from '@material-ui/core/colors/green';
+import red from '@material-ui/core/colors/red';
+
+import Container from '@material-ui/core/Container';
```

Comment:
🌵  Did this pass eslint? I thought it grouped all external imports together (without a new line)

Reply:
Where would eslint fail? Is it set up in the github workflow? Do I need a vscode extension?

Reply:
🤔 I should set it up in a workflow